src/object/srv_obj.c

```diff
@@ -1425,12 +1380,15 @@ ds_obj_punch_local_hdlr(struct obj_punch_in *opi, crt_opcode_t opc,
 		D_GOTO(out, rc = -DER_NOSYS);
 
 	case DAOS_OBJ_RPC_PUNCH:
+	case DAOS_OBJ_RPC_TGT_PUNCH:
```

Comment:
(style)  Possible switch case/default not preceeded by break or fallthrough comment